toolkit/tools/srpmpacker/srpmpacker.go

```diff
@@ -947,14 +939,14 @@ func hydrateFromRemoteSource(fileHydrationState map[string]bool, newSourceDir st
 			}
 		}
 
-		cancelled, internalErr := retry.RunWithExpBackoff(func() error {
+		cancelled, internalErr := retry.RunWithDefaultDownloadBackoff(func() error {
```

Comment:
Nice! Looks like you found and de-duplicated some other copies of the constants; thanks for cleaning them up while you were at it :)

Reply:
I only hope this is not overspecializing the `retry` package's public functions. But you're welcome.

Reply:
I think it's an iterative improvement, now that we've de-duped some magic numbers. If we deem that we need different call sites to have different policy, then we can still do so intentionally but can at least now have a conversation about why its defaults ought to be different.